common/common-client/build.gradle

```diff
@@ -5,6 +5,8 @@ dependencies {
     compile "org.objectweb.proactive:programming-core:${programmingVersion}"
     compile "org.objectweb.proactive:programming-extension-annotation:${programmingVersion}"
 
+    compile "org.keycloak:keycloak-adapter-core:24.0.2"
```

Comment:
can we please check together that this dependency does not add conflicts with our existing libraries ?

Reply:
I already checked and all Keycloak transitive dependencies are green.

- This is the list of dependencies induced by **keycloak-adapter-core:24.0.2**:

```
|    |    +--- org.keycloak:keycloak-adapter-core:24.0.2
|    |    |    \--- org.keycloak:keycloak-crypto-default:24.0.2
|    |    |         +--- org.keycloak:keycloak-core:24.0.2
|    |    |         |    +--- org.keycloak:keycloak-common:24.0.2
|    |    |         |    |    \--- jakarta.activation:jakarta.activation-api:2.1.2
|    |    |         |    +--- com.fasterxml.jackson.core:jackson-core:2.16.1
|    |    |         |    +--- com.fasterxml.jackson.core:jackson-databind:2.16.1
|    |    |         |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.16.1
|    |    |         |    |    \--- com.fasterxml.jackson.core:jackson-core:2.16.1
|    |    |         |    \--- org.eclipse.microprofile.openapi:microprofile-openapi-api:3.1.1
|    |    |         +--- org.keycloak:keycloak-server-spi:24.0.2
|    |    |         +--- org.keycloak:keycloak-server-spi-private:24.0.2
|    |    |         |    \--- org.apache.httpcomponents:httpclient:4.5.14 (*)
|    |    |         +--- org.bouncycastle:bcprov-jdk18on:1.77
|    |    |         \--- org.bouncycastle:bcpkix-jdk18on:1.77
|    |    |              +--- org.bouncycastle:bcprov-jdk18on:1.77
|    |    |              \--- org.bouncycastle:bcutil-jdk18on:1.77
|    |    |                   \--- org.bouncycastle:bcprov-jdk18on:1.77

```

- This is the list of current vulnerabilites in the **Scheduling project**. None of them is induced by Keycloak :

![image](https://github.com/ow2-proactive/scheduling/assets/25662619/3d7461c9-32ea-407a-a097-84981d2f3ac6)